jd-geek-source/src/app/components/test-diplay/test-diplay.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <h4>Testing Getting Data From The Best Buy API</h4>
+
+  <div *ngFor="let product of testResults">
+    <div>
+      <p>
+        <b>Department: </b>{{ product.department }}&nbsp;&nbsp; <b>SKU: </b
```

Comment:
Control spacing with css as opposed to non-breaking spaces for greater consistency, clarity, and responsive flexibility.